samples/apps/copilot-chat-app/webapi/Program.cs

```diff
@@ -99,6 +101,17 @@ private static void AddSemanticKernelServices(this IServiceCollection services,
         // Each API call gets a fresh new SK instance
         services.AddScoped<Kernel>();
 
+        services.AddSingleton<PromptsConfig>(sp =>
+        {
+            string promptsConfigPath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!, "prompts.json");
```

Comment:
Is it possible to do this in the skills' constructors?

Reply:
In services especially, we want to separate the concerns of configuration and instantiation. These PromptSettings should be able to be provided in several different ways and the ChatSkill itself should use what is provided.